scopes/workspace/workspace/eject-conf.cmd.ts

```diff
@@ -14,12 +14,12 @@ type EjectConfOptionsCLI = {
 
 export default class EjectConfCmd implements Command {
   name = 'eject-conf <pattern>';
-  description = 'eject components configuration (create a `component.json` file)';
+  description = 'eject components configuration (create a `component.json` file) - irreversible';
   extendedDescription = `${PATTERN_HELP('eject-conf')}`;
   alias = '';
   group = 'development';
   options = [
-    ['p', 'propagate', 'mark propagate true in the config file'],
+    ['p', 'propagate', 'mark propagate true in the config file'], // @david what does propate do?
```

Comment:
it's a propagation for handling config merges.
meaning - do we want to merge the component configs with things coming from workspace.jsonc or not.
default is - we don't want to merge.